README.md

```diff
@@ -18,14 +18,14 @@ Since Camel-K already does a pretty good job building Camel applications in any
 
 ## How does it work?
 
-Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build". 
+Without going into many details, Camel-K has basically two phases of their build, which I call "Project Assemble" and "Image Build".
 In this first phase, Camel-K reads the Route configuration, assemble the Maven project and run the project build.
 Then it takes the Java application and build into an image.
 
 Camel-K has this concept of "Environment Platform" that based on the type of cluster which is running, can pick the right build feature.
 For example, source to image on OpenShift clusters or Kaniko on Kubernetes.
 
-Kogito might use this first phase to assemble a specific application based on the sources pushed to the cluster. 
+Kogito might use this first phase to assemble a specific application based on the sources pushed to the cluster.
 For example, Serverless Workflow projects can have a slightly different setup than DMN/Decisions one.
 
 For simplicity, I skipped the first phase and assembled the project on a "builder image" for Serverless Workflow projects.
```

Comment:
it might needs a rephrase, it is not clear what exactly "assembled the project on a builder image" means.